code/datums/scenarios/scenario.dm

```diff
@@ -0,0 +1,107 @@
+/singleton/scenario
+	/// The scenario's name. Displayed in the round end report and some other places.
+	var/name = "Generic Scenario"
+	/// The scenario's description. Displayed in the round end report and some other places.
+	var/desc = "A generic scenario that should not be in the rotation."
+	/// What sectors this scenario can spawn in. An empty list is all sectors.
+	var/list/sector_whitelist = list()
+	/// The type of scenario this is. NOT a boolean or a bitfield.
+	var/scenario_type = SCENARIO_TYPE_NONCANON
+
+	/**
+	 * All scenarios happen in an away site, which is manually spawned when the odyssey is picked.
+	 * This away site can have its exoplanet themes defined and then mapped to give the semblance of being on a planet (see the Tret away site).
+	 * Alternatively, without defining exoplanet themes, it can take place in space, for example.
+	 * In the future there will likely be support for no-away site scenarios that take place on the Horizon.
+	 */
```

Comment:
This isn't the correct way to use a DMDoc, make it a standard comment or merge it with the single-line one below if it's significant for the below var